Lib/threading.py

```diff
@@ -647,7 +647,8 @@ def wait(self, timeout=None):
         (or fractions thereof).
 
         This method returns the internal flag on exit, so it will always return
-        True except if a timeout is given and the operation times out.
+        True except if a timeout is given and the operation times out, when it
+        will return `False`.
```

Comment:
Double backticks for rst (single is for md):

```suggestion
        will return ``False``.
```

https://devguide.python.org/documentation/markup/